Tools/python/HgcrocEmulator.py

```diff
@@ -90,6 +90,19 @@ def calculateVoltage(self, electrons) :
         """
 
         return electrons*(0.162/1000.)*(1./self.readoutPadCapacitance)
+
+    def calculateVoltageHcal(self, PE) :
```

Comment:
I'm starting to think that the `calculateVoltage`, `calculateVoltageHcal`, `setThresholdDefaults` and `setThresholdDefaultsHcal` should just be deleted in favor of moving those definitions to our
```python
def EcalHgcrocEmulator() : 
    <subsystem-specific-stuff>
```
function defintions. What do you think?

I can handle copying over that stuff to the ECal and adding that to this PR if you like that idea.

Reply:
I moved everything to the `HcalHgcrocEmulator()` and defined it as a class [here](https://github.com/LDMX-Software/Hcal/commit/deb2a21ef841041d432cc1f15989c813356996c7). Let me know if this is similar to what you were thinking.